logstash-core/lib/logstash/pipeline.rb

```diff
@@ -101,6 +100,10 @@ def ready?
     @ready.value
   end
 
+  def filter_queue_client
+    @filter_queue_client
```

Comment:
It could be but I prefer to leave it as-is. I did originally call this method `filter_queue` but the ivar `@filter_queue_client` but I though it would be more confusing.